sdk/webpubsub/Azure.Messaging.WebPubSub.Client/README.md

```diff
@@ -70,7 +70,14 @@ public async ValueTask<Uri> FetchClientAccessTokenFromServerAsync(CancellationTo
 }
 ```
 
-`WebPubSubServiceClient` can be used to generate the Client Access Uri as well as manage client connections. Find more details in [Azure.Messaging.WebPubSub](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/webpubsub/Azure.Messaging.WebPubSub)
+Features to differentiate `WebPubSubClient` and `WebPubSubServiceClient`.
```

Comment:
this is a little bit confusing, why comparing these 2, how about a graph showing the position of these 2 sdks as what we do in ServiceClient readme

Reply:
Because we mentioned these two clients in the code sample above. There's a graph contains two clients